src/Appwrite/Utopia/Response.php

```diff
@@ -205,6 +205,7 @@ class Response extends SwooleResponse
     public const MODEL_DOMAIN_LIST = 'domainList';
     public const MODEL_VARIABLE = 'variable';
     public const MODEL_VARIABLE_LIST = 'variableList';
+    public const MODEL_VCS = 'vcs';
```

Comment:
I guess this is still missing, just double checking as I don't see the file for this model.